.github/workflows/ci.yml

```diff
@@ -7,11 +7,13 @@ jobs:
     - name: Free disk space
       run: |
         docker system prune -a -f
+        sudo rm -rf /usr/local/lib/android
```

Comment:
question: Would this work without sudo?

Reply:
I saw this and got tripped up by the docker command before it (github docker containers run as root), but then realized this is running on the vm itself, which would require sudo (for removing a directory under `/usr`): https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#administrative-privileges